data/json/obsolete_terrains.json

```diff
@@ -24,7 +24,9 @@
       "ants_larvae",
       "ants_larvae_acid",
       "ants_queen",
-      "ants_queen_acid"
+      "ants_queen_acid",
+      "pond_forest",
+      "pond_swamp"
```

Comment:
Obsolete overmap terrain ids with direction should also be here - `pond_forest_east`, `pond_swamp_north` etc. (see https://github.com/CleverRaven/Cataclysm-DDA/issues/61543).